package.json

```diff
@@ -49,7 +52,7 @@
     "react-is": "^17.0.2",
     "regenerator-runtime": "^0.13.5",
     "simplebar-react": "^2.1.0",
-    "storybook-addon-material-ui": "^0.9.0-alpha.24",
```

Comment:
Why are we dropping this?

Reply:
It was needed to create the theme for storybook. We've now replaced it [MUI ThemeProider](https://github.com/CodeForAfrica/COMMONS-UI/pull/68/files#:~:text=()%3B-,export%20const%20decorators%20%3D%20%5BmuiTheme(%5Btheme%5D),%5D%3B,-const%20size%20%3D)